app/uk/gov/hmrc/pegaproofofconceptstubs/controllers/SubmitPayloadController.scala

```diff
@@ -16,16 +16,24 @@
 
 package uk.gov.hmrc.pegaproofofconceptstubs.controllers
 
+import play.api.Logging
+import play.api.libs.json.JsValue
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
-import play.api.mvc.{Action, AnyContent, ControllerComponents}
+import play.api.mvc.{Action, ControllerComponents}
+import uk.gov.hmrc.pegaproofofconceptstubs.models.Payload
+import uk.gov.hmrc.pegaproofofconceptstubs.models.Payload.formats
+
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.Future
 
 @Singleton()
-class MicroserviceHelloWorldController @Inject() (cc: ControllerComponents)
-  extends BackendController(cc) {
+class SubmitPayloadController @Inject() (cc: ControllerComponents)()
+  extends BackendController(cc) with Logging {
 
-  def hello(): Action[AnyContent] = Action.async { implicit _ =>
-    Future.successful(Ok("Hello world"))
+  val submitPayload: Action[JsValue] = Action.async(parse.json) { implicit request =>
+    withJsonBody[Payload] { payload =>
+      logger.info("[OPS-11581] payload submitted with value: " + payload.data)
```

Comment:
normally we don't bother putting ticket numbers in log messages - what was the reason you did this in SI?

Reply:
when you are searching kibana for the logs, you can tag the ticket number as a string search to find the log outputs easier